assessmentModel/src/commonMain/kotlin/AssessmentExtensions.kt

```diff
@@ -0,0 +1,133 @@
+package org.sagebionetworks.assessmentmodel
+
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.jsonArray
+import org.sagebionetworks.assessmentmodel.survey.AnswerType
+import org.sagebionetworks.assessmentmodel.survey.BaseType
+import org.sagebionetworks.assessmentmodel.survey.ChoiceQuestion
+
+
+/**
+ * Get a flattened list of answers definitions. This provides a list of column names and types
+ * for the results returned by [AssessmentResult.toFlatAnswers]. Column name is defined
+ * as <SectionID>_<NodeID>, for most surveys it will just be the identifier of the question.
+ * The column names match the keys returned by [AssessmentResult.toFlatAnswers]
+ */
+fun Assessment.toFlatAnswersDefinition(): List<AnswerColumn> {
+    return FlatAnswersDefinitionGenerator(this).build()
+}
+
+data class AnswerColumn(
+    val columnName: String,
+    val type: AnswerType,
```

Comment:
```suggestion
    val answerType: AnswerType,
```

Not really any value in using the "type" reserved keyword here since this isn't a json schema property.